pkg/proc/native/threads.go

```diff
@@ -22,26 +22,6 @@ type nativeThread struct {
 	common         proc.CommonThread
 }
 
-// Continue the execution of this thread.
-//
-// If we are currently at a breakpoint, we'll clear it
-// first and then resume execution. Thread will continue until
-// it hits a breakpoint or is signaled.
-func (t *nativeThread) Continue() error {
```

Comment:
Why remove the generic version of this?

Reply:
IMHO it's misleading to have this function defined for freebsd and windows, since in either case it is not possible to resume the execution of individual threads. Also, given how it's used on linux, we should probably replace it with a direct call to (*nativeThread).resume but I didn't want to do it in the middle of a bunch of changes to the freebsd backend (maybe the same is also true for the macos backend, but I haven't looked).

Reply:
Ack, ok that makes sense.